### PR TITLE
Remove CoverageStatsV2.

### PR DESCRIPTION
**Pull request by tmooney**
The non-V2 version is the version used in pipelines; this version was not put into production.  (`AlignmentSummaryV2` was incorporated into the non-V2 version.)

### CODE REVIEW

**Comment by acoffman** [comment]
+1

File: lib/perl/Genome/Model/Tools/BioSamtools/RefCov.pm [hunk 1]
```diff
@@ -86,11 +86,8 @@ sub execute {
     my $output_directory = $self->output_directory;
     my $wingspan = $self->wingspan;
     if ($output_directory) {
-        # TODO: Old CoverageStats required running RefCov with multiple Wingspan valude
-        # In reality, this create redundancy across the ROI.
-
-        # Moving forward CoverageStatsV2 will create multiple BED files.
-        # What do we do with this auto-generated subdirectory....
+        # TODO: CoverageStats requires running RefCov with multiple Wingspan valude
```
Comment (dufeiyu) [inline lib/perl/Genome/Model/Tools/BioSamtools/RefCov.pm]: A typo here: value.

**Comment by dufeiyu** [comment]
+1  just in case, is @jasonwalker80  ok with this removal ?

File: lib/perl/Genome/Model/Tools/BioSamtools/RefCov.pm (hunk 1 above)
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/BioSamtools/RefCov.pm]: Fixed!

**Comment by jasonwalker80** [comment]
+1

**Pull request merged**